client/me/connected-application-item/index.jsx

```diff
@@ -169,7 +169,7 @@ class ConnectedApplicationItem extends React.Component {
 				) }
 				<div>{ this.renderScopeMessage() }</div>
 
-				<h2>{ this.props.translate( 'Access Permissions' ) }</h2>
+				<h2>{ this.props.translate( 'Access permissions' ) }</h2>
```

Comment:
:warning: This change will be queued for retranslation. We'll display the English text until then.